backend/src/main/kotlin/app/ehrenamtskarte/backend/user/KoblenzUser.kt

```diff
@@ -0,0 +1,7 @@
+package app.ehrenamtskarte.backend.user
```

Comment:
maybe should be moved to userdata package, created for the user import endpoint?
or vice versa.
I think keeping both packages can be confusing

Reply:
moved that